draft-ietf-kitten-krb-spake-preauth-04.xml

```diff
@@ -763,8 +750,7 @@ KEY_USAGE_SPAKE_TRANSCRIPT              66
       <t>The following key usage values are assigned for this mechanism:</t>
         <figure>
           <artwork><![CDATA[
-KEY_USAGE_SPAKE_TRANSCRIPT              65
-KEY_USAGE_SPAKE_FACTOR                  66
+KEY_USAGE_SPAKE                         65
```

Comment:
Is 66 going to stay reserved in the registry or get freed up?

Reply:
I will free it up after this is merged.